### DISCUSSION TITLE
Identity configuration on Blazor Bootstrap project

### DISCUSSION THREAD

**Discussion by nhmatsumoto** [vikramlearning/blazorbootstrap#973] [General]
Hello everyone! I’m having trouble setting up Identity within projects that use Blazor Bootstrap. Every time I add a new route for account/login, my login form briefly appears on the page, and then a “not found” message is displayed. Has anyone experienced this issue? Could you help me?

**Comment by gvreddy04** [comment]
@nhmatsumoto Thank you for using BlazorBootstrap. Please share the .NET and Blazor version related information of your project.

**Reply by nhmatsumoto** [reply]
@gvreddy04 Thank you for responding!
Currently, my project is on version 9 of .NET Core.
My version of Blazor Bootstrap is 3.2.0 (server).
I created a basic application to better understand how Blazor Bootstrap works, but after configuring Blazor Bootstrap, the Identity routes stopped working. Unfortunately, the application I created contains some information that I cannot share, but to replicate my scenario, I basically created a .NET Core Blazor Server application in version 9, and in Visual Studio itself, I configured the project to include Identity (generated via scaffold).

**Reply by nhmatsumoto** [reply]
Hello, @gvreddy04 

Recently, I have been trying to solve the problem and found a solution. Apparently, the issue occurs due to the rendering method configured in Blazor Bootstrap. Part of this problem can be resolved by removing @attribute [ExcludeFromInteractiveRouting] from _imports.razor, located in the scaffold-generated components directory. However, removing this configuration leads to other issues related to HttpContext, as it cannot be manipulated during a SignalR connection.

**Reply by nhmatsumoto** [reply]
After many attempts, I discovered that the issue is caused by the Razor rendering mode. The solution can be found at the following link: https://stackoverflow.com/questions/77710968/login-page-of-blazor-8-is-refreshing-in-a-loop
